cli/tests/test_split_command.rs

```diff
@@ -180,6 +181,20 @@ fn test_split_by_paths() {
     A file2
     [EOF]
     ");
+
+    let _ = work_dir.run_jj(["new"]).success();
```

Comment:
nit: You should be able to remove the `let _ = ` (same in some other commits in this PR)